### PR TITLE
Mailet 118 : just review last commit please

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by rouazana** [comment]
:+1: but you should make the test pass :)

**Comment by chibenwa** [comment]
:+1:

File: mailet/base/src/test/java/org/apache/mailet/base/test/MailUtil.java [hunk 1]
```diff
@@ -52,27 +55,23 @@ public static FakeMail createMockMail2Recipients(MimeMessage m) throws ParseExce
...
+        return createMimeMessage(headerName, headerValue, "testmail");
     }
     
-    public static FakeMimeMessage createMimeMessage(String headerName, String headerValue, String subject, int number) throws MessagingException {
+    private static MimeMessage createMimeMessage(String headerName, String headerValue, String subject) throws MessagingException {
         String sender = "test@james.apache.org";
         String rcpt = "test2@james.apache.org";
 
-        FakeMimeMessage mockedMimeMessage = new FakeMimeMessage(number);
+        MimeMessage mockedMimeMessage = new MimeMessage(NO_SESSION);
```
Comment (aduprat) [inline mailet/base/src/test/java/org/apache/mailet/base/test/MailUtil.java]: It's no more a mocked message.

**Comment by rouazana** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Pull request merged**